test/test.c

```diff
@@ -29354,15 +29357,15 @@ void test_JetStreamSubscribePullAsync_Unsubscribe(void)
     args.msgReceived = false;
     args.closed = false;
     args.sum = 0;
+    natsMutex_Unlock(args.m);
+
     jsOptions_Init(&jsOpts);
     jsOpts.PullSubscribeAsync.CompleteHandler = _completePullAsync;
     jsOpts.PullSubscribeAsync.CompleteHandlerClosure = &args;
-    natsMutex_Unlock(args.m);
-    jsFetchRequest lifetime = {
-        .Batch = 100,
-        .Expires = NATS_SECONDS_TO_NANOS(1),
-    };
-    s = js_PullSubscribeAsync(&sub, js, "foo", "dur", _recvPullAsync, &args, &lifetime, &jsOpts, NULL, &jerr);
+    jsOpts.PullSubscribeAsync.MaxMessages = 100;
+    jsOpts.PullSubscribeAsync.TimeoutMillis = 1000;
+
+    s = js_PullSubscribeAsync(&sub, js, "foo", "dur", _recvPullAsync, &args, &jsOpts, NULL, &jerr);
     testCond(s == NATS_OK);
 
     test("Receive 1 message: ");
```

Comment:
This test too:
```
    Test #110: JetStreamSubscribePullAsync_Unsubscribe ...***Failed  Required regular expression not found. Regex=[ALL PASSED
]  0.66 sec
Test using server executable: c:\test\nats-server.exe
Test using server executable: c:\test\nats-streaming-server.exe
Test server version: "nats-server: v2.10.18"

== JetStreamSubscribePullAsync_Unsubscribe ==
#01 Start JS Server: PASSED
#02 Connect: PASSED
#03 Get context: PASSED
#04 Create stream: PASSED
#05 Publish a few messages: PASSED
#06 Subscribe: PASSED
#07 Receive 1 message: FAILED
*** TEST FAILED ***
```

Reply:
ok, will try to repro and read through the code/test, see if I can figure out what's causing these 2. If I can't reproduce anywhere else, I may need to ask you to run an instrumented version on your Windows.

Reply:
@kozlovic Can you please try again? I increased a few more test wait timeouts, hopefully those were the cause of the failures on the slow VM...